BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -2995,6 +3002,7 @@ void PlayFireAnim()
                     }
                     fireState[i].speed = fireAnimSpeed;
                     fireState[i].normalizedTime = Mathf.Repeat(fireState[i].normalizedTime, 1);
+                    Debug.Log("[BDArmory.ModuleWeapon]: playing Fire Anim, i = " + i + "; fire anim " + fireState[i].name);
```

Comment:
This should probably be behind a `if (BDArmorySettings.DEBUG_WEAPONS)` clause.

Reply:
Or commented out/deleted; there's always one debug message that I miss...